### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by NunoMartins21**
# Search Page

## Type of PR

- [x] - feature

## Description

Search page for the website. Capable of searching DAOs, gates and users. Interface based on the Explorer page.

## Checklist

- [x] - Built the page according to the Figmas
- [x] - Reviewed with designers; check the bugs

### CODE REVIEW

File: apps/website/constants/categories.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const CATEGORIES = {
```
Comment (kbooz) [inline apps/website/constants/categories.ts:1]: I would rather put this is in a `dao.ts` and `gate.ts` file 🤔

**Review by kbooz — CHANGES_REQUESTED** [review]
LGTM, only 2 things 😌

File: apps/website/constants/categories.ts (hunk 1 above)
Comment (NunoMartins21) [reply apps/website/constants/categories.ts:1]: Hmm okay, well said. I didn't think about it to be honest, but thanks for the idea

**Review by fbenfraj — APPROVED** [review]
LGTM ✅ 😄

**Comment by Ashishgy** [comment]
No changes are required!

**Pull request merged**